src/components/QueryBuilder.vue

```diff
@@ -168,6 +197,30 @@ $largeViewportWidth: 90em; //~1438px
 	}
 }
 
+.querybuilder__condition-wrapper-below {
+	margin-block-start: 0;
+	border-block-start: none;
+}
+
+.querybuilder__condition-wrapper-above {
+	margin-block-end: 0;
+	border-inline: none;
+	border-block-end: none;
+}
+
+.querybuilder__condition-group-above {
+	border-inline: $border-width-thin $border-style-base $border-color-base-default;
+	background-color: $color-base-80;
+}
+
+.querybuilder__condition-relation-toggle-or {
+	padding-inline-start: $dimension-layout-xsmall;
+}
+
+.querybuilder__condition-relation-toggle {
+	padding-block: $dimension-layout-xsmall;
```

Comment:
I am testing it and I see the and has the spacing and the or is on the left. I think the logic is inverted. 

![image](https://user-images.githubusercontent.com/3005239/106751436-72c2e680-6629-11eb-81d8-482476cb31be.png)

Reply:
Ugh. It was correct but once I was done, I did some refactoring that broke it. It should be fixed now.